packages/utility/agent.ts

```diff
@@ -158,7 +158,14 @@ const Likes = {
   post: (body: {}) => requests.post<Like>('/likes/', body),
   delete: (id: number) => {
     requests.delete<Like>('/likes/' + id.toString());
-  },
+  }
+};
+
+const Notifications = {
+  get: () =>
+    requests.get<Like[]>('/notifications?populate=deep', {
```

Comment:
This is fine for now, but @dbradham we should thrive to remove the need to use `populate=deep`.

Reply:
@chungthuang what is the ideal way to use `populate`? Should we use `populate=*`, or only using specific entities?

Reply:
We should limit to specific entries. With `populate=deep`, strapi will populate nested relations. For example, each notification references a user, a user references a project, a project references an opportunity. `populate=deep` will return all those relations.